ceremonies/2024/run.sh

```diff
@@ -109,3 +109,9 @@ openssl verify \
     "./int-e7.cert.pem" \
     "./int-e8.cert.pem" \
     "./int-e9.cert.pem"
+
+cd ../../hlint
+GOBIN="${TOOLS}/bin" go install ./
+cd -
```

Comment:
This install feels like it should happen inside setup_ceremony_tool

Reply:
I wound up putting this in its own setup function. `setup_ceremony_tool` is gated by an existence check for an executable at `${CEREMONY_BIN_2024}`, and it feels not really correct to have the creation of two executables gated on the existence of just one of them (and gating on the existence of both also introduces weird cases).